.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-- '0.10'
+- '4.2'
```

Comment:
Node 0.10.x is so old that it can't support yarn. Bumping it up because it's ancient.